materializer/nestedfieldselection/README.md

```diff
@@ -0,0 +1,132 @@
+`@materializer` is a great way to seamlessly extend a type with data from another type by executing an operation, sometimes with arguments supplied by the extended type. 
+
+Sometimes the data needed is nested, so @materializer provides the ability to 
+specify a single field selection path.
+
+Assume that the original schema contains documents of type `document` with `content` and `author`, and that `content` has a required identifier provided by any request, and that the query operation only exposes direct access to the document, and not to the individaul `content` and `author`:
+
+```
+type Document {
+   docId: String
+   content(id: String!): Content
```

Comment:
For this snippet it is probably cleaner to remove these `@materializer` , to avoid confusion. Less is more.

Reply:
I removed from the README, but I think I actually need it to make the snippet work